packages/client/components/ScopePhaseAreaAddJiraServer.tsx

```diff
@@ -0,0 +1,88 @@
+import styled from '@emotion/styled'
+import graphql from 'babel-plugin-relay/macro'
+import React from 'react'
+import {useFragment} from 'react-relay'
+import useAtmosphere from '../hooks/useAtmosphere'
+import useMutationProps from '../hooks/useMutationProps'
+import {PALETTE} from '../styles/paletteV3'
+import {ScopePhaseAreaAddJiraServer_meeting$key} from '../__generated__/ScopePhaseAreaAddJiraServer_meeting.graphql'
+import JiraServerSVG from './JiraServerSVG'
+import RaisedButton from './RaisedButton'
+import JiraServerClientManager from '../utils/JiraServerClientManager'
+
+const AddJiraServerArea = styled('div')({
+  display: 'flex',
+  justifyContent: 'center',
+  alignItems: 'center',
+  flexDirection: 'column',
+  height: '100%'
+})
+
+const StyledLink = styled('span')({
+  color: PALETTE.SKY_500,
+  cursor: 'pointer',
+  outline: 0,
+  ':hover, :focus, :active': {
+    color: PALETTE.SKY_600
+  },
+  paddingTop: 24
+})
+
+const AddJiraServerButton = styled(RaisedButton)({
+  whiteSpace: 'pre-wrap'
+})
+interface Props {
+  gotoParabol: () => void
+  meetingRef: ScopePhaseAreaAddJiraServer_meeting$key
+}
+
+const ScopePhaseAreaAddJiraServer = (props: Props) => {
+  const atmosphere = useAtmosphere()
+  const mutationProps = useMutationProps()
+
+  const {gotoParabol, meetingRef} = props
+  const meeting = useFragment(
+    graphql`
+      fragment ScopePhaseAreaAddJiraServer_meeting on PokerMeeting {
+        teamId
+        viewerMeetingMember {
+          teamMember {
+            integrations {
+              jiraServer {
+                auth {
+                  id
+                  isActive
+                }
+                sharedProviders {
+                  id
+                }
+              }
+            }
+          }
+        }
+      }
+    `,
+    meetingRef
+  )
+  const {teamId, viewerMeetingMember} = meeting
+
+  const provider = viewerMeetingMember?.teamMember.integrations.jiraServer?.sharedProviders[0]
```

Comment:
+1 [it looks like](https://github.com/ParabolInc/parabol/blob/8694b37c32b7499061ec7a84ca024e89c2b7304f/packages/server/graphql/types/TeamMemberIntegrations.ts#L30) `jiraServer` is non-nullable so we can drop the `?`